internal/civo/iterator.go

```diff
@@ -0,0 +1,37 @@
+package civo
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/konstructio/dropkick/internal/civo/sdk"
+	"github.com/konstructio/dropkick/internal/compare"
+	"github.com/konstructio/dropkick/internal/outputwriter"
+)
+
+// deleteIterator returns a function that can be used to iterate over resources.
+func (c *Civo) deleteIterator(ctx context.Context) func(sdk.APIResource) error {
```

Comment:
I'm not sure if the function name and description are accurate.

Reply:
Any proposals? 😆

Reply:
For the lack of better recommendations, we'll keep this as-is for the time being. We can always refactor it later.